cider/exec/nextgen/jitlib/base/JITValue.h

```diff
@@ -29,6 +29,16 @@ namespace cider::jitlib {
 class JITValue;
 class JITFunction;
 
+class JITBaseValue {
+ public:
+  JITBaseValue(JITTypeTag type_tag, JITBackendTag backend)
```

Comment:
Hmm, do we want to specify tag type when constructing JIT value? It means each we need to specify ```JITBackendTag``` when new a JITBaseValue. Assume that most case we have only one JIT backend when codegen. Then we can consolidate them in a single place like ```context```. Thoughts?

Reply:
Considering JITFunction have context we needed, so we can create JITTuple through a JITFunction instead create it directly.